src/App.tsx

```
@@ -6,6 +6,8 @@ import './App.css';
 
 function App() {
   const [count, setCount] = useState(0);
+  const x: number = '문자열'; // 타입 오류
+  console.log(x);
```

ESLint

[error] Unexpected console statement.

(no-console)